From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ricardo Pardini <ricardo@pardini.net>
Date: Sat, 3 Jan 2026 21:06:01 +0100
Subject: GENIO: radxa-nio-12l: Enable bind command

After commit 9bc199d7ea5 ("GENIO: board: 350/510/700/1200 EVK: Drop
deprecated usb_ether_init()") USB Ethernet Gadget needs to be manually
binded using the `bind` command. Let's enable that on Radxa NIO-12L.

Fixes: 8d921703e1fc5b446b041a7c81e93ca0455d573c # when nio-12l did not exist yet
Signed-off-by: Ricardo Pardini <ricardo@pardini.net>
---
 configs/genio_1200_radxa_nio_12l_d16_defconfig | 1 +
 configs/genio_1200_radxa_nio_12l_d4_defconfig  | 1 +
 configs/genio_1200_radxa_nio_12l_d8_defconfig  | 1 +
 3 files changed, 3 insertions(+)

diff --git a/configs/genio_1200_radxa_nio_12l_d16_defconfig b/configs/genio_1200_radxa_nio_12l_d16_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_1200_radxa_nio_12l_d16_defconfig
+++ b/configs/genio_1200_radxa_nio_12l_d16_defconfig
@@ -36,6 +36,7 @@ CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
 CONFIG_CMD_ADC=y
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
diff --git a/configs/genio_1200_radxa_nio_12l_d4_defconfig b/configs/genio_1200_radxa_nio_12l_d4_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_1200_radxa_nio_12l_d4_defconfig
+++ b/configs/genio_1200_radxa_nio_12l_d4_defconfig
@@ -36,6 +36,7 @@ CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
 CONFIG_CMD_ADC=y
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
diff --git a/configs/genio_1200_radxa_nio_12l_d8_defconfig b/configs/genio_1200_radxa_nio_12l_d8_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_1200_radxa_nio_12l_d8_defconfig
+++ b/configs/genio_1200_radxa_nio_12l_d8_defconfig
@@ -36,6 +36,7 @@ CONFIG_CMD_NVEDIT_EFI=y
 CONFIG_CMD_NVEDIT_INFO=y
 # CONFIG_CMD_CRC32 is not set
 CONFIG_CMD_ADC=y
+CONFIG_CMD_BIND=y
 CONFIG_CMD_CLK=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_DM=y
-- 
Armbian

